packages/react-components/style/output-components-style.css

```diff
@@ -83,8 +83,8 @@ canvas {
     align-self: center;
 }
 
-.no-data {
-    font-size: 24px;
+.important-message {
+    font-size: 20px;
```

Comment:
Any reason for that change? I didn't find it too big before.

Reply:
Good point. The font size change at line 87 should have been committed in this [earlier commit](https://github.com/theia-ide/theia-trace-extension/pull/574/commits/59d064484cb37a0c8e555d9fa44bd6bea44a8fa0) where I removed the hardcoded style tags to just use the CSS class. In theory there should be no change to how the the text is displayed before and after this PR because one of the style tags was setting the font size to 20. My understanding is that hardcoded style tags override CSS class.